app/assets/stylesheets/works.scss

```diff
@@ -53,3 +53,24 @@
     }
   }
 }
+
+#pagination{
```

Comment:
Hmm, can we move this to a higher level .scss file? Eventually want to apply `ReactPagination` to the `AllArtists` page too, cc @tommypoa